### PR TITLE
Build MicroCerts page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build microcerts hero based on [design](https://app.zeplin.io/project/5fc7a466de658e0cb99276c8/screen/5fd38d07a48d6b903ffb4e1f) at `/cube/microcerts`
- Fixed table column widths
- Hard coded adding 2 passed certifications to see contrast for design review (line 24 and 31 of `/cube/microcerts`)
- Fixed table view on mobile 

## QA

- Check page at: https://ubuntu-com-8951.demos.haus/cube/microcerts
- Check against [design](https://app.zeplin.io/project/5fc7a466de658e0cb99276c8/screen/5fd38d07a48d6b903ffb4e1f)
- Check SCSS and give any feedback on class names 

## Issue / Card

Fixes [#3507](https://github.com/canonical-web-and-design/web-squad/issues/3507)

## Screenshots

image

### CODE REVIEW

File: static/sass/_pattern_cube-progression.scss [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+    %rhombus-badge {
+      background-size: map-get($rhombusSize, "width") * 1px
+        map-get($rhombusSize, "height") * 1px;
+      height: map-get($rhombusSize, "height") * 1px;
+      position: absolute;
+      width: map-get($rhombusSize, "width") * 1px;
+    }
+
+    &__tag {
```
Comment (steverydz) [inline static/sass/_pattern_cube-progression.scss:49]: I think theses should be the full class names rather than concatenated. We've agreed in dev catchup a few times that is the right approach, so in this case it would be `.p-cube-progression__tag`.

Concatenating class names in this way makes them impossible to search for in the codebase and on a file that is taller than your screen they are very hard to follow. It's better to be explicit.

Comment (bethcollins92) [reply static/sass/_pattern_cube-progression.scss:49]: Thanks for your help @steverydz, makes sense I'll change it now 👍🏼

**Comment by cassiocassio** [comment]
I know this is built as visually designed, but I can see this both ways: 

My first impression is that 10 have been completed and 5 remain to be filled in,

Then I look again read the numbers, and see my mistake: that 5 grey inactive tiles are done, and 10 bright lit tiles remain

Anybody else have that impression?

https://www.google.com/url?sa=i&url=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FRabbit%25E2%2580%2593duck_illusion&psig=AOvVaw2ld1BUKAlnsdv062FjKxfb&ust=1608389898379000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCJCzmLbl1-0CFQAAAAAdAAAAABAD

**Comment by anasereijo** [comment]
> I know this is built as visually designed, but I can see this both ways:
> 
> My first impression is that 10 have been completed and 5 remain to be filled in,
> 
> Then I look again read the numbers, and see my mistake: that 5 grey inactive tiles are done, and 10 bright lit tiles remain
> 
> Anybody else have that impression?
> 
> https://www.google.com/url?sa=i&url=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FRabbit%25E2%2580%2593duck_illusion&psig=AOvVaw2ld1BUKAlnsdv062FjKxfb&ust=1608389898379000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCJCzmLbl1-0CFQAAAAAdAAAAABAD

I had the same impression so I had to double-check with Wayne. I suggest making the opposite.

**Comment by bethcollins92** [comment]
@cassiocassio @anasereijo That's easy enough to implement, it's only hard coded atm so once I apply the logic I can do it the opposite way around. Let me know if this is final 👍🏼

File: static/sass/_pattern_cube-progression.scss [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+        map-get($rhombusSize, "height") * 1px;
+      height: map-get($rhombusSize, "height") * 1px;
+      position: absolute;
+      width: map-get($rhombusSize, "width") * 1px;
+    }
+
+    &.is-architecture {
+      @extend %diamond-badge;
+
+      background: url(https://assets.ubuntu.com/v1/8b175afd-Architecture.svg);
```
Comment (barrymcgee) [inline static/sass/_pattern_cube-progression.scss:34]:
```suggestion
      background-image: url(https://assets.ubuntu.com/v1/8b175afd-Architecture.svg);
```

If you're only specifying the background image, you should be explicit otherwise it's possible to override rules further up the cascade using the shorthand.

Comment (barrymcgee) [reply static/sass/_pattern_cube-progression.scss:34]: (The applies throughout this file)

File: templates/cube/microcerts.html [hunk 3]
```diff
@@ -7,16 +7,41 @@
...
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor.
-      </p>
+      <p class="p-heading--4">You have completed {{ passed_courses }} of 15 MicroCertifications</p>
     </div>
-    <div class="col-4"></div>
+    <div class="col-6 p-cube-progression u-hide--medium u-hide--small">
+      <!-- <div class="p-cube-progression is-tag u-align--center u-vertically-center">
+        <span><small>Micro<br /><span>Certifications:</span></small><br /><span class="p-heading--4">{{ passed_courses }}/15</span></span>
+      </div> -->
```
Comment (barrymcgee) [inline templates/cube/microcerts.html:19]: As a general rule, we should not check in commented-out code. However, if there is a good reason for it, you should leave a comment above explaining.

**Review by barrymcgee — CHANGES_REQUESTED** [review]

**Comment by bethcollins92** [comment]
Thanks @barrymcgee made those changes, I was able to delete the commented out code was just waiting on a design confirmation of which option we were going for. (Will remember to leave a comment explaining in the future 👍🏼 )

**Review by barrymcgee — APPROVED** [review]

**Comment by anasereijo** [comment]
@bethcollins92 looking great!!

just a couple of things:

- [ ] I can't see the table when I land on this page (I have a small screen). is it possible to do the following?

Screenshot 2021-01-07 at 15 21 31

Screenshot 2021-01-07 at 15 16 28


- [ ] [check zeplin](https://app.zeplin.io/project/5fc7a466de658e0cb99276c8/screen/5fd38d07a48d6b903ffb4e1f) for new copy 

- [ ]

**Comment by bethcollins92** [comment]
@anasereijo Thanks, I will address this in the coming PR!

**Review by jpmartinspt — APPROVED** [review]
